src/java/fr/paris/lutece/portal/web/system/CacheJspBean.java

```diff
@@ -70,6 +71,10 @@
     private static final String TEMPLATE_CACHE_INFOS = "admin/system/cache_infos.html";
     private static final String PARAMETER_ID_CACHE = "id_cache";
 
+    // Conf
+    private static final String PATH_CONF = "/WEB-INF/conf/";
+    private static final String FILE_PROPERTIES_CONFIG = "config.properties";

```

Comment:
Maybe you shoud not redefine these constants here, but share them with the normal intialisation process. Otherwise, there is a risk these become desynchronized

Reply:
It's already defined in several places as private constants... Not sure if it's useful to declare them as public constants. Maybe in fr.paris.lutece.portal.web.constants.Parameters ? but it looks like this class is for parameters of http requests.